modules/parquet/test/data/files.js

```diff
@@ -1,47 +1,55 @@
+  {supportedJs: false, supportedWasm: false, title: 'bloom_filter', path: 'good/bloom_filter.bin'},
+  {supportedJs: true, supportedWasm: true, title: 'byte_array_decimal', path: 'good/byte_array_decimal.parquet'},
+  {supportedJs: false, supportedWasm: false, title: 'datapage_v2', path: 'good/datapage_v2.snappy.parquet'}, // Doesn't work on parquet-tools
+  {supportedJs: true, supportedWasm: false, title: 'dict', path: 'good/dict-page-offset-zero.parquet'},
+  {supportedJs: true, supportedWasm: true, title: 'fixed_length_decimal', path: 'good/fixed_length_decimal.parquet'},
+  {supportedJs: true, supportedWasm: true, title: 'fixed_length_decimal_legacy', path: 'good/fixed_length_decimal_legacy.parquet'},
+  {supportedJs: false, supportedWasm: false, title: 'hadoop_lz4_compressed', path: 'good/hadoop_lz4_compressed.parquet'},
+  {supportedJs: false, supportedWasm: false, title: 'hadoop_lz4_compressed_larger', path: 'good/hadoop_lz4_compressed_larger.parquet'},
+  {supportedJs: true, supportedWasm: true, title: 'int32_decimal', path: 'good/int32_decimal.parquet'},
+  {supportedJs: true, supportedWasm: true, title: 'int64_decimal', path: 'good/int64_decimal.parquet'},
+  {supportedJs: true, supportedWasm: false, title: 'list_columns', path: 'good/list_columns.parquet'},
+  {supportedJs: true, supportedWasm: false, title: 'nation', path: 'good/nation.dict-malformed.parquet'}, // Partially works, issue with indices for dictionary values.
+  {supportedJs: true, supportedWasm: false, title: 'nested_lists', path: 'good/nested_lists.snappy.parquet'},
+  {supportedJs: true, supportedWasm: false, title: 'nested_maps', path: 'good/nested_maps.snappy.parquet'},
+  {supportedJs: false, supportedWasm: false, title: 'nested_structs', path: 'good/nested_structs.rust.parquet'},
+  {supportedJs: true, supportedWasm: false, title: 'nonnullable', path: 'good/nonnullable.impala.parquet'},
+  {supportedJs: true, supportedWasm: false, title: 'nullable', path: 'good/nullable.impala.parquet'},
+  {supportedJs: true, supportedWasm: false, title: 'nulls', path: 'good/nulls.snappy.parquet'},
+  {supportedJs: true, supportedWasm: false, title: 'repeated_no_annotation', path: 'good/repeated_no_annotation.parquet'},
+  {supportedJs: false, supportedWasm: true, title: 'single_nan', path: 'good/single_nan.parquet'},
+
+  {supportedJs: false, supportedWasm: true, title: 'data_index_bloom_encoding_stats', path: 'good/data_index_bloom_encoding_stats.parquet'},
+  {supportedJs: false, supportedWasm: false, title: 'delta_binary_packed', path: 'good/delta_binary_packed.parquet'},
+  {supportedJs: false, supportedWasm: false, title: 'delta_byte_array', path: 'good/delta_byte_array.parquet'},
+  {supportedJs: false, supportedWasm: false, title: 'delta_encoding_optional_column', path: 'good/delta_encoding_optional_column.parquet'},
+  {supportedJs: false, supportedWasm: false, title: 'delta_encoding_required_column', path: 'good/delta_encoding_required_column.parquet'},
```

Comment:
These are new files from https://github.com/apache/parquet-testing that didn't previously exist here.